tests/conftest.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+from datetime import datetime
+
+import pytest
+from dateutil import tz
+
+from arrow import arrow, factory, formatter, locales, parser
+
+
+@pytest.fixture(scope="class")
+def utcnow_fixture(request):
```

Comment:
Nitpicky comment, but I think we should remove the `_fixture` prefix since the decorator above and the `@pytest.mark.usefixtures` calls make it obvious.

Reply:
done, also did some renaming to have consistency.